.gitignore

```diff
@@ -31,3 +31,6 @@ app/assets/builds/
 .yardoc
 
 .devcontainer/output
+
+# Bust A Gem VSCode extension
+TAGS
```

Comment:
I really like this extension which allows _go to definition_ for symbols/etc... for bundled gems. It generates a ctags file locally.

Reply:
Nice! I use solargraph which is able to leverage the Gems yardoc; but we should definitely support folks who prefer CTAGS